### ISSUE TITLE
[Feature] Investigate Hatch's monorepo features for a repo to hold Qadence extra features

### ISSUE THREAD

**Issue by RolandMacDoland** [pasqal-io/qadence-libs#27]
From @jpmoutinho pointer to [Hatch doc](https://hatch.pypa.io/1.9/config/hatch/).

This concerns the development of a landing place for:

- [ ] Qadence-protocols
- [ ] Qadence-libs
- [ ] MLTools

**Comment by mlahariya** [comment]
Hey @sungwoo-pasqal , @RolandMacDoland , can we please set up a meeting to discuss this in the next week?

**Comment by sungwoo-pasqal** [comment]
Yeah, this is a good opportunity to dive deeper into the details of Hatch. I’ll discuss it with Roland.

**Comment by RolandMacDoland** [comment]
@sungwoo-pasqal is this resolved ?

**Comment by sungwoo-pasqal** [comment]
@RolandMacDoland Yes it is resolved
